editor/src/uuiui/styles/utopi-color-helpers.ts

```diff
@@ -1,47 +1,34 @@
 import ChromaWrongTypes from 'chroma-js'
 const Chroma = ChromaWrongTypes as any
 
-export interface UtopiColor {
+export class UtopiColor {
   /**
-   * css-compatible rgba string: `rgba(255,0,255,1)` format
+   * css variable associated with this color: var(--variable-name) format
    */
-  value: string
-
+  value: string = 'var(--utopitheme-not-set)'
   /**
-   * Opacity or Alpha. Value goes from 0 to 100
+   * css-compatible rgba string: `rgba(255,0,255,1)` format
    */
-  o: (value: number) => UtopiColor
-}
+  cssValue: string = '--utopitheme-not-set'
 
-type ColorHex = string
+  constructor(baseColor: string, path?: string) {
+    const hexWithAlpha = Chroma(baseColor).css('rgba')
```

Comment:
It made sense when we were catching every color value so that there were no unnecessary duplicates (like an hsl and rgb version of the same color). But now I don't think it is needed. In fact, now that I think about it, this class can be converted back to a single function.

Reply:
Removed UtopiColor class and `createUtopiColor` is now just a very simple function

Reply:
Nice!